ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -8,7 +8,9 @@
 # of our control -- the only version of CP4D that works with MAS 8.7 is v4.0.7, but we can't install that
 # from the IBM operator catalog at present, much more work is needed in this collection to support the operating
 # model of the Cloud Pak for Data team :/
-cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
+cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') | default(lookup('env', 'CPD_RELEASE_VERSION'), true) | default('4.0.9', true)  }}"
```

Comment:
Will we keep CP4D 4.5 as an exception after merge this PR and eventually change the default to it I presume?

Reply:
That's what I believe we'll do... probably when MAS 8.9 is released, CPD 4.5 will become the default.
@durera is that a fair assumption?